src/chartsFactory.jsx

```diff
@@ -61,7 +61,13 @@ module.exports = function (chartType, Highcharts){
     },
 
     render: function (){
-      let props = this.props;
+      let {
```

Comment:
Can we leverage the fact that those properties are described in propTypes and use thing somehow instead having them duplicated here?